libdnf/module/module_sack_impl.hpp

```diff
@@ -89,6 +89,7 @@ class ModuleSack::Impl {
 
 inline const std::vector<std::unique_ptr<ModuleItem>> & ModuleSack::Impl::get_modules() {
     add_modules_without_static_context();
+    create_module_solvables();
```

Comment:
I am sorry I've got a feeling that it will results that solvables will be created redundantly. I guess we need to distinguish whether solvable was already created or not. May be it is possible to use the same trick like with modules without static context. I propose to create additional tmp vector then the workflow will be as follows:
```modules_without_static -> modules_without_solvable -> fully_initialized_modules```
Modules with static context will go directly to modules_without_solvable.

The second approach could be without tmp vector.
Or solvable can be created directly when static_context is preset. For items without static context, solvable can be created directly when static context is calculated

Reply:
Thanks. The additional vector has a problem that when determining the static contexts, I'd have to go through both `fully_initialized_modules` and `modules_without_solvable`, when seemed a bit more complicated. So, I found it better to create the solvables when modules are added to the `modules` vector.